test/unit/notifications.server.test.ts

```diff
@@ -0,0 +1,23 @@
+import sendNotification from '../../app/lib/notifications.server';
+
+describe('Notification server lib function test', () => {
+  const email: string = 'dev@starchart.invalid';
+  const subject: string = 'Sample subject';
+  const text: string = 'Sample text';
+
+  test('sending email notification using sendNotification() ', async () => {
+    const response = await sendNotification(email, subject, text);
+    expect(response).toBe(true);
+  });
+
+  test('verifying MailHog received test email notification via SendNotification() ', async () => {
+    const response = await fetch(`http://localhost:8025/api/v2/search?kind=to&query=${email}`);
+    const responseToJSON = await response.json();
+    const mostRecentMessage = responseToJSON.items[0].Content;
```

Comment:
This is dangerous, since we may add more tests later that mean there are other items in the mailbox.  Don't depend on the state of external systems, order, etc.

A better approach would be to use `expect.arrayContaining()`, which allows you to define a partial array, and make sure that the larger array includes this.

See https://vitest.dev/api/expect.html#expect-arraycontaining and https://vitest.dev/api/expect.html#expect-objectcontaining.